.circleci/config.yml

```diff
@@ -27,7 +24,7 @@ jobs:
           - npm-cache-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
           - npm-cache-{{ arch }}-{{ .Branch }}
           - npm-cache-{{ arch }}
-      - run: bin/setup
+      - run: mix local.hex --force && mix local.rebar --force && mix deps.get
```

Comment:
Why not `bin/setup`?

Reply:
There is way more setup needed on a dev machine than on CI. I think at this point I'm going to remove the CI stuff in the `bin/setup` and pus another commit.